ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto

```diff
@@ -15,6 +15,9 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Command Service is able to correlate submitted commands with completion data, identify timeouts, and return contextual
 // information with each tracking result. This supports the implementation of stateless clients.
+//
+// Note that submitted commands MAY produce a completion event as well. For example, the participant MAY choose to
```

Comment:
I don't understand what this sentence brings. Submitted commands usually lead to completion events

Reply:
This was explicitly stated for the command submission service but not for the command service (where it's less obvious, since it's synchronous).

Reply:
The use of that capitalized **MAY** suggests to me that completion events stemmed from submitted commands via the CommandService are the exception, not the norm. And the follow-up sentence doesn't clarify the idea. Maybe rewording it, if I understood the purpose right, might help: `Note that submitted commands produce completion events as well, even in some cases when commands get rejected (e.g. the participant MAY choose to produce a completion event for a rejection of a duplicate command)`

Reply:
@tudor-da Reworded accordingly — let me know what you think.

Reply:
Thanks lgtm 👍